pkg/infrastructure/attribute_manager.go

```diff
@@ -132,6 +136,19 @@ func (m *AttributeManager) GetAttributeValue(name, instance string) (string, boo
 	return "", false
 }
 
+// returns the normalized-to-scale value of an infrastructure attribute based on the attribute and instance names
+func (m *AttributeManager) GetNormalizedAttributeValue(name, instance string, scale *taxonomy.RangeType) (string, error) {
```

Comment:
I don't think scale should be passed to any of the methods in AttributeManager. Given name+instance, or any other key, scale can be calculated directly from the attribute structure.

Reply:
Done